contributing.md

```diff
@@ -0,0 +1,37 @@
+# What this repository is
+This repository contains sources of the Introduction to Python course.
+
+# Contributing
+We love contributions!
+We are happy to see fixes of the existing bugs and tasks and tests mistakes.
```

Comment:
We welcome fixes for existing bugs, as well as corrections for task and test mistakes.